src/types/operating_hours_type.py

```diff
@@ -2,6 +2,15 @@
 
 from src.types.food_station_type import FoodStationType
 
+class CollegetownEventType(ObjectType):
```

Comment:
what's the reasoning for this change?

Reply:
we were just putting in empty strings for most of the attributes in the main eatery event type, since I made a new ctown eatery type, might as well make types for the hours/events too